scripts/utils.js

```diff
@@ -137,3 +138,47 @@ export async function startBatch(
 export function sleep(ms) {
     return new Promise((res, _) => setTimeout(res, ms));
 }
+
+/// DEBUG UTILS ///
+/**
+ * Execute a given function if we are in debug mode
+ * @param {Function} func - a function we want to execute
+ * @param {...any} args - The arguments to pass to the function
+ */
+export function debugEval(func, ...args) {
+    if (debug) {
+        func(...args);
+    }
+}
```

Comment:
I don't like this function, it's just going to lead to heisenbugs. Perhaps remove the `export`